critiquebrainz/frontend/views/test/test_review.py

```diff
@@ -264,21 +264,23 @@ def test_place_review_pages(self):
         self.assert200(response)
         self.assertIn("A great place.", str(response.data))
 
-    def test_hide_unhide(self):
+    @mock.patch('critiquebrainz.db.user.User.is_admin')
+    def test_hide_unhide(self, is_user_admin):
         # create a review by self.user and check it's not hidden
         review = self.create_dummy_review()
         self.assertEqual(review["is_hidden"], False)
 
         # make self.hacker as current user
         self.temporary_login(self.hacker)
+        is_user_admin.return_value = False
```

Comment:
It's be better to test this by adding the username to the admin list in the config, instead of mocking this methods. 
With CB-385 this will become easier.

Reply:
Sure, for now let's leave this the way it is for now. I'll add a note to CB-385 to fix this as a part of it.